behind-the-veil-siteroot/server/main.js

```diff
@@ -30,11 +30,18 @@ import "/imports/api/collections/messages.js";
 import "/imports/api/methods/messages.js";
 import "/imports/api/publications/messages.js";
 
+
 import "/server/mailer.js"
 import {checkBookingsEveryMidnight} from "./background.js"
 
 // file containing creds for mail server
 import {mailUrl, fromUser} from "./secrets.js"
+import "/imports/api/collections/reviews.js";
+import "/imports/api/methods/reviews.js";
+import "/imports/api/publications/reviews.js";
+
+// file in the same folder containing creds for mail server
+// import {emailUser, emailPass, mailUrl, fromUser} from "./secrets.js"
```

Comment:
Whats this for btw?

Reply:
not sure, i don't remember doing this, might have been a mistake during rebase or idk